rpm/0003-Fix-last-modified-time-for-new-exception.patch

```diff
@@ -0,0 +1,36 @@
+From fa7cb35486bb7c54282e82b035b27e5ae77b0491 Mon Sep 17 00:00:00 2001
+From: Boris Shmarin <b.shmarin@omp.ru>
+Date: Thu, 23 Sep 2021 15:50:37 +0300
+Subject: [PATCH] Set the correct last modified time when creating an exception
+ for a recurring event.
+
+---
+ src/calendar.cpp | 5 ++++-
+ 1 file changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/src/calendar.cpp b/src/calendar.cpp
+index 2a4fef1d0..2d1e5de22 100644
+--- a/src/calendar.cpp
++++ b/src/calendar.cpp
+@@ -5,6 +5,7 @@
+   SPDX-FileCopyrightText: 2000-2004 Cornelius Schumacher <schumacher@kde.org>
+   SPDX-FileCopyrightText: 2003-2004 Reinhold Kainhofer <reinhold@kainhofer.com>
+   SPDX-FileCopyrightText: 2006 David Jarvie <djarvie@kde.org>
++  SPDX-FileCopyrightText: 2021 Boris Shmarin <b.shmarin@omp.ru>
+ 
+   SPDX-License-Identifier: LGPL-2.0-or-later
+ */
+@@ -584,7 +585,9 @@ Incidence::Ptr Calendar::createException(const Incidence::Ptr &incidence, const
+     }
+ 
+     Incidence::Ptr newInc(incidence->clone());
+-    newInc->setCreated(QDateTime::currentDateTimeUtc());
++    QDateTime current = QDateTime::currentDateTimeUtc();
```

Comment:
Not sure it's mandatory but you may declare this `current` variable `const`.

Reply:
Done.